libraries/cms/html/html.php

```diff
@@ -552,81 +543,112 @@ protected static function includeRelativeFiles($folder, $file, $relative, $detec
 	/**
 	 * Write a `<img>` element
 	 *
-	 * @param   string   $file      The relative or absolute URL to use for the src attribute.
-	 * @param   string   $alt       The alt text.
-	 * @param   mixed    $attribs   String or associative array of attribute(s) to use.
-	 * @param   boolean  $relative  Path to file is relative to /media folder (and searches in template).
-	 * @param   mixed    $path_rel  Return html tag without (-1) or with file computing(false). Return computed path only (true).
+	 * @param   array  $options   options to be added to the stylesheet.
 	 *
 	 * @return  string
 	 *
-	 * @since   1.5
+	 * @since   3.6
 	 */
-	public static function image($file, $alt, $attribs = null, $relative = false, $path_rel = false)
+	public static function image($options = array())
 	{
-		if ($path_rel !== -1)
+		// For B/C. Convert old function signature.
+		if (!is_array($options))
```

Comment:
Add a JLog statement for use of the old signature.  https://github.com/joomla/joomla-cms/blob/staging/libraries/cms/application/cms.php#L988